### DISCUSSION TITLE
refetchInterval is not work

### DISCUSSION THREAD

**Discussion by riskers** [wevm/wagmi#4296] [Q&A] [answered]
I want to refetch when used `useBalance`:

```ts
const {data} = useBalance({
  address: '0x4557B18E779944BFE9d78A672452331C186a9f48',
  query: {
    refetchInterval: 5000
  }
})

console.log('data', data)
```

But `refetchInterval` is not work.

Why? What else needs to be done?

**Comment by tmm** [comment, marked as answer]
Check your browser devtools network tab. There should be a new request every five seconds, but the hook will only update if `data` changes (so you won't see the `console.log` unless that happens).
